### PR TITLE
Move TPA to PID profile

### PR DESCRIPTION
**Pull request by haslinghuis**
Changes for https://github.com/betaflight/betaflight/pull/11779

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by limonspb — DISMISSED** [review]

**Comment by haslinghuis** [comment]
Do we like to move the UI part:

image

**Comment by limonspb** [comment]
> Do we like to move the UI part:
> 
> image

of course! :)
And sonarcloud says there is a bug?

**Review by limonspb — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Pull request merged**
